mail_restrict_auto_follower/wizard/mail_wizard_invite.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp import models, api
+
+
+class MailWizardInvite(models.TransientModel):
+    _inherit = 'mail.wizard.invite'
+
+    @api.multi
+    def add_followers(self):
+        obj = self.with_context(allow_follower_subscription=True)
```

Comment:
if you implement https://github.com/OCA/social/pull/22/files#r45470005, this file can go away